.github/workflows/build_all.yml

```diff
@@ -893,6 +893,88 @@ jobs:
           SLACK_ICON_EMOJI: ":heavy_check_mark:"
           SLACK_COLOR: "#00FF00"
           SLACK_FOOTER: ' '
+  build-deploy-arm:
```

Comment:
This still seems to fail with:
"Decoding ARM AGW artifacts JSON has failed:  "

The state on magma ci is not showing up as well.